src/io/mod.rs

```diff
@@ -101,20 +116,23 @@ impl Io {
 
         if Self::is_gzip_path(p) {
             Ok(Box::new(BufReader::with_capacity(self.buffer_size, MultiGzDecoder::new(buf))))
+        } else if Self::is_zstd_path(p) {
+            Ok(Box::new(BufReader::with_capacity(self.buffer_size, ZstdDecoder::new(buf).unwrap())))
```

Comment:
Please remove the `unwrap()` and replace with `?` or if necessary a `map_err()?`

Reply:
Done